docs/src/main/asciidoc/security-keycloak-authorization.adoc

```diff
@@ -169,6 +169,12 @@ quarkus.keycloak.policy-enforcer.enable=true
 
 NOTE: By default, applications using the `quarkus-oidc` extension are marked as a `service` type application (see `quarkus.oidc.application-type`). This extension currently supports only such `service` type applications.
 
+=== Configuring protected routes
```

Comment:
Hi @stefnotch I've had another look at this doc snippet, sorry, it is just a few lines, but I have one more proposal on how to change it. At the moment it feels it is out of context as it is placed in a section where the quickstart steps are described, showing how to protect the routes using this extension. Another minor bit is that this extension is also configured in `application.properties` :-)

IMHO it will read better if you add a single line pointing to the alternative approach at the very end of the introduction section at the top of the document:

```
To configure the protected routes using the `@RolesAllowed` annotation or Quarkus HTTP security policies, check the link:security-openid-connect[Using OpenID Connect Adapter to Protect JAX-RS Applications] guide and the link:security#authorization-of-web-endpoints-using-configuration[Security guide].
```

This way the quickstart solution section will be just about this quickstart.

Thanks

Reply:
I see, it is indeed a bit out of place. I wanted to put it in its own section, to make it very visible.

Would putting it in its own section at the end, also be a reasonable option? Maybe right after `Mapping Protected Resources`, since it's basically a recommendation to look at those tutorials after the quickstart?

Reply:
@stefnotch, sure, how would you name this dedicated section though ? `Next Steps` ? May be `Additional Information` ? This last one might be fine, what do you think ?

Reply:
An alternative name, in line with the heading above it, would be `Configuring Protected Resources`. But `Additional Information` also seems fine.

Reply:
Hi @stefnotch, but this extension is also about protecting the resources with the relevant configuration. That said, lets settle around your `Configuring Protected Resources` but highlight somehow this section is about the alternatives, `More About Configuring Protected Resources` or `Other Options for Configuring Protected Resources` or `Configuring Protected Resources without Keycloak`, etc, find the line you like please :-).

Please also review the proposed amendment to the section text itself but tweak as needed

Thanks, and sorry it takes us awhile to finalize this PR :-)

CC @pedroigor

Reply:
Ok, I updated it according to your suggestions.